packages/utils/ast-utils.ts

```diff
@@ -626,14 +626,35 @@ function parseMerge(j: JSCodeshift, ast: Node, value: string, action: string): b
 				type: "MemberExpression"
 			},
 			operator: "=",
-			right: j.callExpression(j.identifier("merge"), [j.identifier(value), exportsDecl.pop()]),
+			right: j.callExpression(j.identifier("merge"), [j.identifier(configIdentifier), exportsDecl.pop()]),
 			type: "AssignmentExpression"
 		};
 		(p.value as Node).body[bodyLength - 1] = newVal;
 		return false; // TODO: debug later
 	}
+
+	function addMergeImports(ast: Node, mergeImports: string[]) {
+		ast.find(j.Program).filter((program: Node): boolean => {
+			mergeImports.forEach(
+				(imp: string): void => {
+					if ((program.value as Node).body.indexOf(imp) === -1){
```

Comment:
Currently checking whether in the current iteration these two imports are added or not. Similar to `parseTopScope` which also not check this.

I want to make a better checking if a particular package is already imported or not by iterating over all `CallExpression`s and checking values of `argument` where `callee` is `require`. This same function could also be used for `parseTopScope` for better checking when imports are pushed to the topScope.
Should I do that in this PR itself on open a new one for this? asking as scope of that change is larger that that of this PR.

Reply:
Take it in this PR, but could you rephrase your naming conventions such that it makes sense

Reply:
Like?

Reply:
What is an `imp` ? what does it do?

Reply:
Short for `import`, import is a reserved keyword and can't use as a variable name.

It is one of the imports in `mergeImports` array.

Reply:
exactly, just getting you on the right track here from a different perspective

Reply:
Okay 😂, any other naming you felt bad ?